pkg/controllers/raycluster_webhook_test.go

```diff
@@ -124,7 +127,7 @@ func TestRayClusterWebhookDefault(t *testing.T) {
 
 	t.Run("Expected required service account name for the head group", func(t *testing.T) {
 		test.Expect(validRayCluster.Spec.HeadGroupSpec.Template.Spec.ServiceAccountName).
-			To(Equal(validRayCluster.Name+"-oauth-proxy"),
+			To(Equal(oauthServiceAccountNameFromCluster(validRayCluster)),
```

Comment:
Personally I would prefer using specific string here instead of function.
Using function can potentially hide expected value, as you can't be sure if the test passed because expected value is `validRayCluster.Name+"-oauth-proxy"` or `validRayCluster.Name+"-oauth-proxy-ecaffa62"`

Reply:
And below too.